contracts/VotingMachines/Governance.sol

```diff
@@ -0,0 +1,509 @@
+pragma solidity ^0.4.18;
+
+import "../controller/Reputation.sol";
+import "./IntVoteInterface.sol";
+import "../universalSchemes/UniversalScheme.sol";
+import "./GovernanceFormulasInterface.sol";
+
+
+
+contract Governance is IntVoteInterface,UniversalScheme,GovernanceFormulasInterface {
+    using SafeMath for uint;
+
+    enum ProposalState { Closed, Executed, NoneBoosted, Boosted }
+
+    struct Parameters {
+        Reputation reputationSystem; // the reputation system that is being used
+        uint noneBoostedVoteRequierePercentage; // the absolute vote percentages bar.
+        uint noneBoostedVotePeriodLimit; //the time limit for a proposal to be in an absolute voting mode.
```

Comment:
Misspelled, should be nonBoostedVotePeriodLimit?

Reply:
Update at a6ba131